docs/content/components/content/xloader/xloader.mdx

```diff
@@ -1,31 +1,66 @@
 ---
 title: XLoader
-caption: Should be used for loading state.
+caption: Used to indicate the progress of the task.
 ---
 
-The `<XLoader>` component, is a simple `<SVG>` that should be used for loading states.
-It should be displayed when the user is waiting for an upcoming event.
+The `<XLoader>` component visually indicates that a task is currently in progress. It can be used as a placeholder for content that is loading or to block interaction with the screen while waiting for a process to complete, such as data fetching or form submission. This ensures users are aware that something is happening in the background, preventing confusion and enhancing the user experience.
```

Comment:
> It can be used as a placeholder for content that is loading or to block interaction with the screen while waiting for a process to complete, such as data fetching or form submission. This ensures users are aware that something is happening in the background, preventing confusion and enhancing the user experience.

I think thats also written in the usage part.

Reply:
updated. not sure what else to write here honestly 🙈